src/tests/ftest/object/same_key_different_value.yaml

```diff
@@ -0,0 +1,15 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+      test_machines:
+        - boro-A
+timeout: 180
+server_config:
+   name: daos_server
+pool:
+   createmode:
+     mode: 511
+   createset:
+     setname: daos_server
+   createsize:
+     size: 1073741824
```

Comment:
is that the pool size? seems an overkill for this simple test..

Reply:
Pool size has been reduced in updated yaml file